docs/weave/weave-api-spec/03-querying.md

```diff
@@ -0,0 +1,191 @@
+---
+id: weave-query-spec
+title: Querying Weave
+sidebar_label: Querying Weave
+---
+
+In this section querying weave for data will be explained. Before diving into the details, Weave internals such as __buckets__ and __paths__ must be explained.
```

Comment:
Sometimes you're writing _weave_ and other times _Weave_ (first letter upper or lower). I think it is good to pick one style.

Reply:
Good point. I think the capital letter propagates the magnificence of `Weave`  :crossed_swords: